SwiftProtobuf.podspec

```diff
@@ -16,6 +16,7 @@ Pod::Spec.new do |s|
   s.cocoapods_version = '>= 1.7.0'
 
   s.source_files = 'Sources/SwiftProtobuf/**/*.swift'
+  s.resource_bundle = {'SwiftProtobuf' => ['Sources/SwiftProtobuf/PrivacyInfo.xcprivacy']}
```

Comment:
btw - Most things I've seen name this with `*_Privacy` (so `SwiftProtobuf_Privacy`), I'm not sure if it really matters or if you want to follow that convention.

Reply:
I really don't know.   The format I've used here seems to be commonly recommended in a few places.  I've yet to get any information from folks at Apple, though.